### DISCUSSION TITLE
Variant calling excercise

### DISCUSSION THREAD

**Discussion by IceGreb** [sandbox-bio/sandbox.bio#6] [Q&A] [answered]
Greetings again @robertaboukhalil ,

So it seems in the tutorial step 3/9 of Variant Calling, the exercise criteria is not working properly. I complete step 1/2 and when i proceed with the command for step 2/2 , the previous step gets unchecked... 

Again, thank you for sandbox.bio :smiley:

**Comment by robertaboukhalil** [comment, marked as answer]
Hello again @IceGreb! Thanks for reporting this, it should now be fixed--refreshing the page should do the trick, but you might need to clear your cache (but just fyi if you're not logged in, clearing the cache wipes out your progress).

**Reply by IceGreb** [reply]
Works like a charm! thanks again for the rapid response! 
Is there a command i can use to reset the files intended to be used in the tutorials? (i seem to have broken the header of a bam file :cry: ... i mean besides clearing my cache)

**Reply by robertaboukhalil** [reply]
Great! Yes, I've been meaning to add such a command. But in the meantime, you can delete the broken file with `rm abc.bam`, wait a second (so the file system changes are synced to your browser's local storage), and refresh the page. sandbox.bio should detect the missing file and recreate it for you.
